lib/services/workflow-api-service.js

```diff
@@ -67,6 +69,9 @@ function workflowApiServiceFactory(
             var context = configuration.context || {};
             if (node) {
                 context = _.defaults(context, { target: node.id });
+                lookupService.nodeIdToProxy(node.id).then(function(proxy) {
+                    context.proxy = proxy;
```

Comment:
It would need to be something like:

```
if (node) {
    context = _.defaults(context, { target: node.id });
    return lookupService.nodeIdToProxy(node.id).then(function(proxy) {
        context.proxy = proxy;
        return self.createActiveGraph(
                    definition, configuration.options, context, configuration.domain, true)
    });
}
return self.createActiveGraph(
            definition, configuration.options, context, configuration.domain, true);
```